### ISSUE TITLE
feat: Being able to approve browser extension requests on apple watch

### ISSUE THREAD

**Issue by air2** [twofas/2fas-ios#104]
### Feature description

Currently the only option to approve requests is in the 2FAS app.  It would be really nice to be able to acknowledge (already configured, like with a token configuration assigned to the requesting website) on your watch. It's totally fine to do the configuring step on the phone itself, but once that is done, it would be nice to be able to use the watch to approve.

### Motivation

Currently I have to dig up my phone, click on the notification, which opens the app, and click approve. Especially digging up my phone takes some effort (Yeah I am lazy :) ), but my watch is on my wrist, so it would be much much faster if I could just acknowledge this request from there.

It would make this awesome app just that bit more awesome.

### Acknowledgements

- [X] This issue is not a duplicate of an existing feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Chefski** [comment]
Literally this. I remember when Microsoft Authenticator allowed signing in with the watch and it was my favourite thing ever. Then they removed it 😭

This feature is the only thing I've wanted ever since I started using 2fas

**Comment by IceHawk66** [comment]
This would be super awesome. Please implement this.

**Comment by sendmebits** [comment]
I would love to see this feature as well. Currently I get the notification on the Apple Watch which is really just a distraction as you have to go to the iPhone to action (Approve) it.

**Comment by adocyn** [comment]
Hi, taking into account current architecture where security is a priority, we would need to add pairing to BE on the watch side. Not an easy feat. We're checking available options.

**Comment by Cl8vM1nd** [comment]
Totally agree! Will help a lot and speed up the process. Taking phone every time is a nightmare.

**Comment by portalhacker** [comment]
2FAS is awesome!

Just like @Chefski this is my only feature request since I started using it all the the time and just bought a Watch.

@adocyn can we support this feature specifically through donations?

**Comment by SolidSonicTH** [comment]
The Watch functionality seems to be pretty limited in general.

Is there any active development on the Watch side of things?
